scripts/deploy/.releaserc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "branches": [
+    "master",
+    {
+      "name": "*",
+      "prerelease": "${name}"
+    }
+  ],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
```

Comment:
I suspect this won't work as expected because it'll get the changelog for all commits on the master branch, not just commits that impact this directory. What do you think?